up.sh

```diff
@@ -35,6 +35,19 @@ if [ -s $CERT_DOMAIN.fullchain.pem ] && ( [ ! -s $CERT_SUB.fullchain.pem ] || ha
   sed -i -e "s,${CERT_SUB},${CERT_DOMAIN},g" ./postfix/conf.d/main.cf
 fi
 
+if dig +short DS "${DOMAIN}" | grep -q .; then
+  # Enable DNSSEC in Postfix
+  sed -i \
+      -e 's/^smtp_dns_support_level.*/smtp_dns_support_level = dnssec/' \
+      ./postfix/conf.d/main.cf
+
+  # If entry does not exist, append it
+  grep -q "^smtp_dns_support_level" ./postfix/conf.d/main.cf || \
```

Comment:
@chrisblech  I think there is a confusion here.

[`smtp_dns_support_level = dnssec`](https://www.postfix.org/postconf.5.html#smtp_dns_support_level) is a client-side setting (meaning that SL will try and lookup domain names using DNS and check if the DNS records are trust worthy using DNSSEC.

This seems unrelated to the fact that SL might be exposed from a domain whose DNS Zone is protected using DNSSEC.

I think I’m seeing warnings from many outgoing sessions in the postfix logs around DNSSEC failures which seems to confirm my assumption.

In any case, I think we need to remove that as setting this up seems complex and it is suggested in the docs that we need our own nameserver as well, which seems overkill for self-hosting SL.


What do you think?